### PR TITLE
Modifications for concourse pipeline

### PR DESCRIPTION
**Pull request by rgoswami-ch**
Added a version file and made changes in the make file.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,38 +1,44 @@
...
+artifact_name   := company-accounts-library
+version         := "unversioned"
 
 .PHONY: all
 all: build
 
 .PHONY: clean
 clean:
-	mvn clean
+    mvn clean
```
Comment (Jata2k) [inline Makefile:9]: These need to stay as tabs as it's a Makefile. Ordinarily yes we'd go the other way

File: version [hunk 2]
```diff
@@ -0,0 +1 @@
+1.1
```
Comment (Jata2k) [inline version:1]: I'd bump this to `1.2`

File: Makefile [hunk 3]
```diff
@@ -1,38 +1,44 @@
...
+    mvn jar:jar deploy:deploy
 
 .PHONY: sonar
 sonar:
-	mvn sonar:sonar
+    mvn sonar:sonar
+
+.PHONY: sonar-pr-analysis
+sonar-pr-analysis:
+    mvn sonar:sonar -P sonar-pr-analysis
```
Comment (Jata2k) [inline Makefile:44]: You'll need to extend a ch parent pom if you want to use this profile

Comment (rafahop) [reply Makefile:44]: I'll merge #10 then

**Pull request merged**